core/templates/dev/head/pages/exploration_editor/simple_editor_tab/StatesToQuestionsService.js

```diff
@@ -174,6 +183,13 @@ oppia.factory('StatesToQuestionsService', [
       // current set of states to a list of questions.
       getQuestions: function() {
         return getQuestions();
+      },
+      getQuestionTypeLabel: function(interactionId) {
+        for( var i = 0; i < SUPPORTED_INTERACTION_TYPES.length; i++) {
```

Comment:
Spacing: `for (var`

Or maybe better to use forEach, since you don't really need the 'i'.